src/main/java/org/prebid/server/handler/SetuidHandler.java

```diff
@@ -121,6 +132,18 @@ private static UsersyncMethodType preferredUserSyncType(Usersyncer usersyncer) {
                 .get(); // when usersyncer is present, it will contain at least one method
     }
 
+    private static void validateUsersyncers(List<Usersyncer> usersyncers) {
+        final List<String> cookieFamilyNames = usersyncers.stream()
```

Comment:
What is the point of this variable? You end up creating another List with no reason.

Reply:
Collections.frequency(...) doesn't allow streams as input. But okay, I will not create such local variable.

Reply:
It left from some previous idea. Will be purged, nevermind.